apptail/instance.go

```diff
@@ -97,6 +119,12 @@ FORLOOP:
 				err = t.Wait()
 				break FORLOOP
 			}
+			currentOffset, err := t.Tell()
```

Comment:
>does trying to t.Tell() here ever make sense?

how elese can we get the current offset?

> Also, does t.Tell() even return errors correctly?

t.Tell() returns whatever tail.file.Seek would return while it's grabbing the offset.
Are  you suggesting to intercept the tailFile process if t.Tell() returns an error?